aws/resource_aws_s3_bucket_object.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsS3BucketObject() *schema.Resource {
 				ValidateFunc: validation.StringInSlice(s3.ObjectCannedACL_Values(), false),
 			},
 
+			"bucket_key_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
If the S3 Bucket has this configuration enabled, but the configuration is not provided to the Object resource, does this return a plan difference? It might be good to add a covering acceptance test for this situation. If it does return a plan difference, we should add `Computed: true` here to signal that the value may set outside the resource configuration.

```suggestion
			"bucket_key_enabled": {
				Type:     schema.TypeBool,
				Optional: true,
				Computed: true,
			},
```

Reply:
Oh, good catch! Test added which failed before adding the `Computed` field, and passes now.